--- openembedded-core/meta/conf/bitbake.conf	2015-03-25 19:24:29.527397951 +0100
+++ conf/bitbake.conf	2015-03-27 14:08:59.892316597 +0100
@@ -1,3 +1,4 @@
+# Automatically generated file with command make patches. Do not edit!
 ##################################################################
 # Standard target filesystem paths.
 ##################################################################
@@ -13,9 +14,9 @@
 BASELIB_powerpc64 = "lib64"
 
 # Path prefixes
-export base_prefix = ""
-export prefix = "/usr"
-export exec_prefix = "/usr"
+export base_prefix = "/share/Apps/openpch"
+export prefix = "/share/Apps/openpch"
+export exec_prefix = "/share/Apps/openpch"
 
 # Base paths
 export base_bindir = "${base_prefix}/bin"
@@ -526,10 +527,11 @@
 TARGET_LINK_HASH_STYLE ?= "${@['-Wl,--hash-style=gnu',''][d.getVar('LINKER_HASH_STYLE', True) != 'gnu']}"
 
 export LDFLAGS = "${TARGET_LDFLAGS}"
-export TARGET_LDFLAGS = "-Wl,-O1 ${TARGET_LINK_HASH_STYLE}"
-#export TARGET_LDFLAGS = "-L${STAGING_DIR_TARGET}${libdir} \
-#                         -Wl,-rpath-link,${STAGING_DIR_TARGET}${libdir} \
-#                         -Wl,-O1"
+# Use rpath for openpch
+#export TARGET_LDFLAGS = "-Wl,-O1 ${TARGET_LINK_HASH_STYLE}"
+export TARGET_LDFLAGS = "-L${STAGING_DIR_TARGET}${libdir} \
+                         -Wl,-rpath-link,${STAGING_DIR_TARGET}${libdir} \
+                         -Wl,-O1"
 
 # Which flags to leave by strip-flags() in bin/build/oebuild.sh ?
 ALLOWED_FLAGS = "-O -mcpu -march -pipe"
@@ -605,7 +607,7 @@
 
 # Use pseudo as the fakeroot implementation
 PSEUDO_LOCALSTATEDIR ?= "${WORKDIR}/pseudo/"
-PSEUDO_PASSWD ?= "${STAGING_DIR_TARGET}"
+PSEUDO_PASSWD ?= "${STAGING_DIR_TARGET}${base_prefix}"
 export PSEUDO_DISABLED = "1"
 #export PSEUDO_PREFIX = "${STAGING_DIR_NATIVE}${prefix_native}"
 #export PSEUDO_BINDIR = "${STAGING_DIR_NATIVE}${bindir_native}"
